backend/src/utils/web_utils.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from googlesearch import search
+import requests
+from bs4 import BeautifulSoup
+from src.prompts import PromptEngine
+from src.utils import Config
+
+
+logger = logging.getLogger(__name__)
+config = Config()
+
+engine = PromptEngine()
+
+def search_urls(search_query) -> list:
+    logger.info(f"Searching the web for: {search_query}")
+    urls = []
+    try:
+        for url in search(search_query, num_results=5):
+            urls.append(url)
+    except Exception as e:
+        logger.error(f"Error during web search: {e}")
+    return urls
+
+
+def scrape_content(url, limit=100000) -> list:
```

Comment:
I think this function returns a string not a list?

Reply:
> I think this function returns a string not a list?

Or better, a json object/string

Reply:
Changed the return type to JSON.